[2024-05-14T09:02:15.303+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-14T09:02:15.312+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline_jobs104.data_lake manual__2024-05-14T09:02:04.753439+00:00 [queued]>
[2024-05-14T09:02:15.316+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline_jobs104.data_lake manual__2024-05-14T09:02:04.753439+00:00 [queued]>
[2024-05-14T09:02:15.316+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-05-14T09:02:15.322+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): data_lake> on 2024-05-14 09:02:04.753439+00:00
[2024-05-14T09:02:15.326+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=65) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-14T09:02:15.328+0000] {standard_task_runner.py:63} INFO - Started process 67 to run task
[2024-05-14T09:02:15.328+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'data_pipeline_jobs104', 'data_lake', 'manual__2024-05-14T09:02:04.753439+00:00', '--job-id', '108', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline_jobs104.py', '--cfg-path', '/tmp/tmp5gvg64vv']
[2024-05-14T09:02:15.330+0000] {standard_task_runner.py:91} INFO - Job 108: Subtask data_lake
[2024-05-14T09:02:15.352+0000] {task_command.py:426} INFO - Running <TaskInstance: data_pipeline_jobs104.data_lake manual__2024-05-14T09:02:04.753439+00:00 [running]> on host 64cecfa2af77
[2024-05-14T09:02:15.431+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_pipeline_jobs104' AIRFLOW_CTX_TASK_ID='data_lake' AIRFLOW_CTX_EXECUTION_DATE='2024-05-14T09:02:04.753439+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-14T09:02:04.753439+00:00'
[2024-05-14T09:02:15.432+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-14T09:02:15.438+0000] {logging_mixin.py:188} INFO - data_lake !!!
[2024-05-14T09:02:15.439+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-14T09:02:15.439+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-14T09:02:15.444+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=data_pipeline_jobs104, task_id=data_lake, run_id=manual__2024-05-14T09:02:04.753439+00:00, execution_date=20240514T090204, start_date=20240514T090215, end_date=20240514T090215
[2024-05-14T09:02:15.472+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-14T09:02:15.492+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-14T09:02:15.494+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
